### PR TITLE
AB#37905 Interns/jerryxihe/37905 create chat sample query

### PR DESCRIPTION
**Pull request by jerryxihe**
[Graph Explorer test site for this branch](https://developer.microsoft.com/en-us/graph/graph-explorer?devx-api=https://graphexplorerapi.azurewebsites.net&org=LokiLabs&branchName=interns/jerryxihe/37905-create-chat-sample-query)

ADO Board Link: [ADO Task #37905](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_sprints/taskboard/GI21%20-%20Graph%20Explorer/Intern%20GitHub/Y21-S/03?workitem=37905)

Action item: After this is merged, use this sample query to verify that the Modify Permissions tab works with updated permissions ([ADO Task #38303](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_sprints/taskboard/GI21%20-%20Graph%20Explorer/Intern%20GitHub/Y21-S/04?workitem=38303)).

### CODE REVIEW

File: sample-queries/sample-queries.json [hunk 1]
```diff
@@ -1393,6 +1393,23 @@
...
+        "requestUrl": "/v1.0/chats",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-post?view=graph-rest-1.0&tabs=http",
+        "headers": [
+            {
+                "name": "Content-Type",
+                "value": "application/json"
+            }
+        ],
+        "postBody": "{\r\n    \"chatType\": \"oneOnOne\",\r\n    \"members\": [\r\n    {\r\n    \"@odata.type\": \"#microsoft.graph.aadUserConversationMember\",\r\n    \"roles\": [\"owner\"],\r\n    \"user@odata.bind\": \"https://graph.microsoft.com/v1.0/users('{your-user-id}')\"\r\n    },\r\n    {\r\n    \"@odata.type\": \"#microsoft.graph.aadUserConversationMember\",\r\n    \"roles\": [\"owner\"],\r\n    \"user@odata.bind\": \"https://graph.microsoft.com/v1.0/users('{user-id}')\"\r\n    }\r\n    ]\r\n    }",
+        "tip": "This query requires your user id and another user's id.  To find your user id & another user's id, you can run: 1) GET https://graph.microsoft.com/v1.0/me/ 2) GET https://graph.microsoft.com/v1.0/users",
```
Comment (acchiang) [inline sample-queries/sample-queries.json:1410]: Should we be concerned about localization here? How do these sample queries deal with localization (aka translation) for tips?

File: sample-queries/sample-queries.json [hunk 2]
```diff
@@ -270,7 +270,7 @@
         "humanName": "get a user's presence ",
         "requestUrl": "/beta/users/{user-id}/presence",
         "docLink": "https://docs.microsoft.com/en-us/graph/api/cloudcommunications-getpresencesbyuserid?view=graph-rest-beta&tabs=http",
-        "tip": "This query requires a user id and Presence.Read.All permissions. To find user id, you can run:  GET https://graph.microsoft.com/v1.0/users",
+        "tip": "This query requires a user id and Presence.Read.All permissions. To find a user id, you can run:  GET https://graph.microsoft.com/v1.0/users",
```
Comment (acchiang) [inline sample-queries/sample-queries.json:273]: will localization be affected with this change?

**Review by acchiang — COMMENTED** [review]
some questions :D

**Review by lhao03 — APPROVED** [review]
lgtm! it works! but same questions as allison

**Review by gracet9182 — APPROVED** [review]
lgtm!

File: sample-queries/sample-queries.json (hunk 1 above)
Comment (jerryxihe) [reply sample-queries/sample-queries.json:1410]: When this fork gets merged into [microsoftgraph/microsoft-graph-devx-content](https://github.com/microsoftgraph/microsoft-graph-devx-content), Elena Ostrovskaia will be automatically pinged about changes in this file, and she will handle editing of the translated sample query files.

File: sample-queries/sample-queries.json (hunk 2 above)
Comment (jerryxihe) [reply sample-queries/sample-queries.json:273]: https://github.com/LokiLabs/microsoft-graph-devx-content/pull/7#discussion_r660819126

**Comment by acchiang** [comment]
lgtm! thanks for touching base

**Pull request merged**